### ISSUE TITLE
Make appointment slots 2 hours instead of 1 hour.

### ISSUE THREAD

**Issue by rkim00** [rubyforgood/circulate#607]

**Comment by rkim00** [comment]
Converting to issue from note per @CTLvivek . Vivek, is this something that's an idea being kicked around or something that folks definitely want to do? And what do we see as the benefit / pain point resolved? This might be something that isn't too hard to do. FYI @jim .

**Comment by CTLvivek** [comment]
*After writing my response below, I see that this change has already been implemented* 

Good point, Richard! I was trying to get into the habit of transferring things from Slack to github and I overshot on that one.  It was mostly a point of discussion but there wasn't any dissenting opinions so I jumped the gun and put it in as a done deal.

The initial reason it was proposed was to reduce people calling or emailing because they were running a bit early or late.  Because our appointments are mostly for planning/preparation, they aren't absolute and the thought was that 2 hours windows might make that more obvious.  Another benefit that I could envision is making the scheduling window easier to navigate.
